cmd/kube-controller-manager/app/batch.go

```diff
@@ -46,6 +47,17 @@ func startCronJobController(ctx ControllerContext) (http.Handler, bool, error) {
 	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "batch", Version: "v1beta1", Resource: "cronjobs"}] {
 		return nil, false, nil
 	}
+	if features.DefaultMutableFeatureGate.Enabled(features.CronjobController2) {
+		cj2c, err := cronjob.NewController2(ctx.InformerFactory.Batch().V1().Jobs(),
+			ctx.InformerFactory.Batch().V1beta1().CronJobs(),
+			ctx.ClientBuilder.ClientOrDie("cronjob-controller"),
+		)
+		if err != nil {
+			return nil, true, fmt.Errorf("error creating CronJob controller: %v", err)
```

Comment:
all the logs and every message should explicitly mention that it's CronJob controller v2, to know which is running, or failed running.

Reply:
added